docs/examples/user_types/ruby/user_policy.polar

```diff
@@ -1,26 +0,0 @@
-# simple-start
-# Internal users have access to both the
-# internal and customer dashboards
-allow(actor: InternalUser, "view", "internal_dashboard");
-allow(actor: InternalUser, "view", "customer_dashboard");
-
-# Customers only have access to the customer dashboard
-allow(actor: Customer, "view", "customer_dashboard");
-# simple-end
-
-# rbac-start
-# Internal users can access the accounts dashboard if
-# they are an account manager
-allow(actor: InternalUser, "view", "accounts_dashboard") if
-    actor.role = "account_manager";
```

Comment:
The Ruby version of this policy used idiomatic Ruby method calls (no parens for a no-arg call)